pkg/timeline/model/list.ts

```diff
@@ -0,0 +1,67 @@
+import type { AccountID } from '../../accounts/model/account.js';
+import type { ID } from '../../id/type.js';
+
+export type ListID = ID<List>;
+export interface CreateListArgs {
+  id: ListID;
+  title: string;
+  publicity: 'PUBLIC' | 'PRIVATE';
+  ownerId: AccountID;
+  memberIds: AccountID[];
+  createdAt: Date;
+}
```

Comment:
```suggestion
export type CreateListArgs = Readonly<{
  id: ListID;
  title: string;
  publicity: 'PUBLIC' | 'PRIVATE';
  ownerId: AccountID;
  memberIds: readonly AccountID[];
  createdAt: Date;
}>;
```
オブジェクト作成のために使うパラメータの意図であれば, 読み取ることしなしないので `readonly` を付けておきたいです.